Dockerfile

```
@@ -1,11 +1,16 @@
-FROM gcr.io/distroless/static:nonroot
+FROM golang:1.21 as build
 
-# `nonroot` coming from distroless
-USER 65532:65532
+WORKDIR /go/src/app
+COPY . .
+
+RUN go mod download
+RUN GOOS=darwin GOARCH=arm64 CGO_ENABLED=0 go build -o /go/bin/datum
```

```suggestion
RUN CGO_ENABLED=1 go build -o /go/bin/datum
```

You need `CGO_ENABLED` for the sqlite driver